server/Helpers/signIn.js

```diff
@@ -3,6 +3,10 @@ import jwt from "jsonwebtoken";
 // todo: reduce this time and implement refresh token
 const ACCES_TOKEN_EXPIRATION = 86400; // 24h in secondes
 
+export const deleteAccessTokenCookie = (res) => {
```

Comment:
Since we don't use function of `Helpers/signIn.js` only in sing-in maybe we should rename it ?
or put it int `Helpers/auth.js` with `checkToken` ?
I let you decide what you think is best

Reply:
Moved deleteAccessTokenCookie to SignOut.js